### PR TITLE
use put_view over render/4 due deprication in phoenix

### PR DESCRIPTION
**Pull request by fatboypunk**
to get ready before phoenix 1.4

### CODE REVIEW

**Review by sn3p — DISMISSED** [review]

File: test/ex_cell/controller_test.exs [hunk 1]
```diff
@@ -6,14 +6,22 @@ defmodule ExCell.ControllerTest do
   end
 
   test "cell/2 with controller" do
-    assert ExCell.Controller.cell(%{}, :mock_cell) === [%{}, :mock_cell, "template.html", []]
+    assert ExCell.Controller.cell(%{private: %{}}, :mock_cell) === [
+             %{private: %{phoenix_view: :mock_cell}},
```
Comment (sn3p) [inline test/ex_cell/controller_test.exs:10]: What does the `private` part do exactly?

Comment (fatboypunk) [reply test/ex_cell/controller_test.exs:10]: it's what put_view does, it calls put_private_view on Plug.Conn and sets the chosen view as a value under the key phoenix_view in the private map on the conn.

Comment (fatboypunk) [reply test/ex_cell/controller_test.exs:10]: https://github.com/phoenixframework/phoenix/blob/v1.3.3/lib/phoenix/controller.ex#L435

**Review by sn3p — APPROVED** [review]

**Comment by jessedijkstra** [comment]
🎉

**Pull request merged**